akka-actor/src/main/scala/akka/dispatch/sysmsg/SystemMessage.scala

```diff
@@ -237,12 +237,12 @@ private[akka] final case class Supervise(child: ActorRef, async: Boolean) extend
  * INTERNAL API
  */
 @SerialVersionUID(1L)
-private[akka] case class Watch(watchee: InternalActorRef, watcher: InternalActorRef) extends SystemMessage // sent to establish a DeathWatch
+private[akka] final case class Watch(watchee: InternalActorRef, watcher: InternalActorRef) extends SystemMessage // sent to establish a DeathWatch
 /**
  * INTERNAL API
  */
 @SerialVersionUID(1L)
-private[akka] final case class Unwatch(watchee: ActorRef, watcher: ActorRef) extends SystemMessage // sent to tear down a DeathWatch
+private[akka] final case class Unwatch(watchee: InternalActorRef, watcher: InternalActorRef) extends SystemMessage // sent to tear down a DeathWatch
```

Comment:
I would have to test, but I'm quite sure it will not interoperate with older format.
Is it too late to change ? 2.4 must be wire-compatible with 2.3 ? I'd like to see this in protobuf too !

Otherwise, the 2nd commit may be dropped or changed to something else (it's merely "cosmetic", not functional)